templates/caching-service.json

```diff
@@ -25,6 +25,19 @@
             "name": "${APPLICATION_NAME}"
          }
       },
+      {
+         "kind": "Secret",
+         "apiVersion": "v1",
+         "metadata": {
+            "name": "${APPLICATION_NAME}"
+         },
+         "stringData" : {
+            "application-user" : "${APPLICATION_USER}",
+            "application-password" : "${APPLICATION_USER_PASSWORD}",
+            "management-user" : "${MANAGEMENT_USER}",
+            "management-password" : "${MANAGEMENT_USER_PASSWORD}"
```

Comment:
TBH I think we do not need management user/password. Only application level credentials.

Reply:
I was on the fence with this TBH. I guess if we allow management access it brakes the encapsulation of the caching/shared-memory service and allows the user todo more than they should. I'll remove it